learning/courses/quantum-business-foundations/quantum-technology.mdx

```diff
@@ -10,31 +16,23 @@ By the end of this module, you should be able to:
 
-### Qiskit Runtime on IBM Cloud
+### Qiskit Runtime
 
 Qiskit Runtime is a portable, secure, containerized architecture that runs quantum programs on a classical computing unit tightly integrated with the quantum processor. Qiskit Runtime allows the quantum computer to become a part of any computing environment to accelerate computation—similar to a graphics processing unit (GPU)—and handles the job orchestration and data transfer to the quantum processing unit, maximizing efficiency.
 
-Qiskit and Qiskit Runtime make it simple to quickly orchestrate programs across compute resources on the cloud. IBM builds middleware for quantum to maximize the performance of quantum applications running across parallelized, cloud-based, quantum and classical computational resources. This middleware includes the **circuit knitting** toolbox and our **quantum serverless** architecture.
-
-* The Circuit knitting toolbox allows developers to cut large circuits into smaller circuits suitable for current quantum computers. Circuit knitting uses classical computation to take on some of the computational burden of a quantum circuit to exceed what we can achieve with either alone. Additional tools help reconstruct circuits with classical resources and stitch together final results.
+Qiskit and Qiskit Runtime make it simple to quickly orchestrate programs across compute resources on the cloud. IBM builds middleware for quantum to maximize the performance of quantum applications running across parallelized, cloud-based, quantum and classical computational resources. This middleware includes the circuit knitting toolbox and our quantum serverless architecture.
 
-* The Quantum Serverless library is a multi-cloud orchestration toolkit to connect elastic classical resources with the IBM Qiskit Runtime service. Quantum serverless allows developers to focus on the code, rather than resource provisioning.
+The [circuit cutting Qiskit addon](https://pypi.org/project/circuit-knitting-toolbox/) allows developers to cut large circuits into smaller circuits suitable for current quantum computers. Circuit knitting uses classical computation to take on some of the computational burden of a quantum circuit to exceed what we can achieve with either alone. Additional tools help reconstruct circuits with classical resources and stitch together final results.
 
-We demonstrated a 120x speedup in simulating molecules thanks to a host of improvements, including the ability to run quantum programs entirely on the cloud with Qiskit Runtime. Read more: [IBM Quantum delivers 120x speedup of quantum workloads](https://research.ibm.com/blog/120x-quantum-speedup).
+[Quantum Serverless](/docs/guides/serverless) is a multi-cloud orchestration toolkit to connect elastic classical resources with the IBM Qiskit Runtime service. Quantum serverless allows developers to focus on the code, rather than resource provisioning.
```

Comment:
```suggestion
[Quantum Serverless](/docs/guides/serverless) is a multi-cloud orchestration toolkit to connect elastic classical resources with the IBM Qiskit Runtime service. Quantum Serverless allows developers to focus on the code, rather than on resource provisioning.
```